docs/modules/plugins/pages/plugin-azure-resource-manager.adoc

```diff
@@ -71,6 +71,25 @@ resourceGroups/sample-resource-group/providers/Microsoft.KeyVault/vaults/sample-
 When I get Azure resource with identifier `resourceGroups/TEST-RG/providers/Microsoft.KeyVault/vaults/KEY-VAULT-NAME` using API version `2021-10-01` and save it to scenario variable `key-vault-properties`
 ----
 
+==== Get information about Azure resource using Azure resource URL
```

Comment:
Shouldn't you deprecate the old step?

Reply:
nope, I would keep the old step, because it's simpler for majority of the resources